scripts/bootstrap.sh.in

```diff
@@ -122,6 +122,9 @@ install_salt_minion() {
 }
 
 configure_salt_minion_local_mode() {
+    "$SALT_CALL" --file-root=/srv/scality/metalk8s-@@VERSION/salt \
```

Comment:
For the commit message: you described the problem (which is good!), but you could also describe how you solved it (by sync'ing the Salt files earlier in bootstrap process, IIUC).

Reply:
Commit message updated